content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -91,8 +88,8 @@ spec:
 {{< note >}}
 
 **Note:** It is recommended that RuntimeClass write operations (create/update/patch/delete) be
-restricted to the cluster administrator. This is typically the default. See [Authorization
-Overview](https://kubernetes.io/docs/reference/access-authn-authz/authorization/) for more details.
+restricted to the cluster administrator. This is typically the default.
+See [Authorization Overview](https://kubernetes.io/docs/reference/access-authn-authz/authorization/) for more details.
 
```

Comment:
should we need change this?

Reply:
This change is necessary if you have editor to do the syntax highlighting. I think it is needed.

Reply:
can't follow